colour/models/rgb/rgb_colourspace.py

```diff
@@ -689,7 +693,8 @@ def __str__(self) -> str:
         Use Derived NPM    : False
         Use Derived NPM -1 : False
         """
-
+        if self._derived_matrix_XYZ_to_RGB is None:
```

Comment:
Yes we should initialize to none explicitly. Good review point. This was entirely because I kept getting frustrated with load times 😂 there are a lot of RGB spaces that get initialized the first time colour is imported.